src/geocat/comp/climatology.py

```diff
@@ -322,3 +352,140 @@ def month_to_season(
     # Therefore, we must filter out the NaNs
     return means.sel(
         {time_coord_name: means[time_coord_name].dt.month == months[1]})
+
+
+def calendar_average(
+        dset: typing.Union[xr.DataArray, xr.Dataset],
+        freq: str,
+        time_dim: str = None,
+        climatology: bool = False,
+        calendar: str = None) -> typing.Union[xr.DataArray, xr.Dataset]:
```

Comment:
```suggestion
        calendar: str = None) -> typing.Union[xr.DataArray, xr.Dataset]:
```

I would delete the calendar kwarg. it seems useless. What happens if a user provides a calendar that is not the input calendar?

Reply:
> I would delete the calendar kwarg


👍🏽 I concur...

Reply:
This has been removed in the latest commit.